ports/geckolib/gecko_bindings/structs_debug.rs

```diff
@@ -2760,6 +2764,84 @@ pub enum SheetParsingMode {
 pub struct HandleRefPtr<T> {
     pub mHandle: T,
 }
+pub enum nsAttrAndChildArray { }
+pub enum nsChildContentList { }
+pub enum nsCSSSelectorList { }
+pub enum nsDOMAttributeMap { }
+pub enum nsIAnimationObserver { }
+pub enum nsIDOMElement { }
+pub enum nsIDOMNodeList { }
+pub enum nsIEditor { }
+pub enum nsIFrame { }
+pub enum nsINodeList { }
+pub enum nsNodeSupportsWeakRefTearoff { }
+pub enum nsNodeWeakReference { }
+pub enum nsDOMMutationObserver { }
+pub enum ServoNodeData { }
+pub enum EventListenerManager { }
+pub enum BoxQuadOptions { }
+pub enum ConvertCoordinateOptions { }
+pub enum DOMPoint { }
+pub enum DOMQuad { }
+pub enum DOMRectReadOnly { }
+pub enum Element { }
+pub enum Text { }
+pub enum TextOrElementOrDocument { }
+pub enum DOMPointInit { }
+#[repr(u32)]
+#[derive(Debug, Copy, Clone, Eq, PartialEq, Hash)]
+pub enum NodeFlags {
```

Comment:
I'm curious, what causes the name NodeFlags to be generated here, given the enum in nsINode.h has no name?